components/questions/selectedQuestions/SelectedQuestions.tsx

```diff
@@ -61,32 +42,37 @@ class SelectedQuestionsComponent extends React.Component<SelectedQuestionsProps>
             </div>
             <QuestionsList
               className={styles.appQuestionsList}
-              selectedQuestionIds={this.props.selectedQuestionIds}
+              selectedQuestionIds={selectedQuestionIds}
               questions={{ isLoading: false, data: { data: questions } }}
               selectable={false}
               unselectable={true}
-              toggleQuestion={this.toggleQuestion}
+              toggleQuestion={toggleQuestion}
             />
           </div>
         </section>
       </AnimateHeight>
     );
-  }
-
-  toggleQuestion = (questionId: Question['id']) => {
-    this.props.deselectQuestion(questionId);
-  };
-}
-
-const mapStateToProps = (state: AppState) => {
-  return {
-    selectedQuestionsWithCategories: getSelectedQuestionsWithCategories(state),
-    selectedQuestionIds: getSelectedQuestionsIds(state),
-    areAnyQuestionSelected: getAreAnyQuestionSelected(state),
   };
-};
 
-const mapDispatchToProps = { deselectQuestion: ActionCreators.deselectQuestion };
+  const toggleQuestion = useCallback(
+    (questionId: Question['id']) => {
+      dispatch(() => ActionCreators.deselectQuestion(questionId));
+    },
+    [dispatch]
+  );
 
-const SelectedQuestions = connect(mapStateToProps, mapDispatchToProps)(SelectedQuestionsComponent);
-export default SelectedQuestions;
+  if (areAnyQuestionSelected) {
+    return (
+      <TransitionGroup
+        appear={false}
+        enter={false}
+        className={styles.selectedQuestionsContainer}
+        component="div"
+      >
+        {renderSelectedQuestionsList()}
+      </TransitionGroup>
+    );
+  } else {
+    return <NoQuestionsSelectedInfo />;
+  }
+});
```

Comment:
```suggestion
}, (prevProps, nextProps) => isEqual(prevProps, nextProps));
```
I don't remember why, byt it was there before
```
shouldComponentUpdate(nextProps: Readonly<SelectedQuestionsProps>): boolean {
  return !isEqual(this.props, nextProps);
}
```